#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_pgrep.1.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cleanup pgrep manual page #282168

@DPATCH@
diff -urNad procps-3.2.7~/pgrep.1 procps-3.2.7/pgrep.1
--- procps-3.2.7~/pgrep.1	2006-06-26 11:32:41.000000000 +1000
+++ procps-3.2.7/pgrep.1	2006-06-26 11:44:01.000000000 +1000
@@ -7,30 +7,35 @@
 pgrep, pkill \- look up or signal processes based on name and other attributes
 
 .SH SYNOPSIS
-pgrep [\-flvx] [\-d \fIdelimiter\fP] [\-n|\-o] [\-P \fIppid\fP,...] [\-g \fIpgrp\fP,...]
-.br
-	[\-s \fIsid\fP,...] [\-u \fIeuid\fP,...] [\-U \fIuid\fP,...] [\-G \fIgid\fP,...]
-.br
-	[\-t \fIterm\fP,...] [\fIpattern\fP]
+.HA
+.na
+\fBpgrep\fR [\fB\-flvx\fR] [\fB\-d\ \fIdelimiter\fR] [\fB\-n\fR|\fB\-o\fR] \
+[\fB\-P\ \fIppid\fR,...] [\fB\-g\ \fIpgrp\fR,...] [\fB\-s\ \fIsid\fR,...] \
+[\fB\-u\ \fIeuid\fR,...] [\fB\-U\ \fIuid\fR,...] [\fB\-G\ \fIgid\fR,...] \
+[\fB\-t\ \fIterm\fR,...] [\fIpattern\fR]
 
-pkill [\-\fIsignal\fP] [\-fvx] [\-n|\-o] [\-P \fIppid\fP,...] [\-g \fIpgrp\fP,...]
-.br
-	[\-s \fIsid\fP,...] [\-u \fIeuid\fP,...] [\-U \fIuid\fP,...] [\-G \fIgid\fP,...]
-.br
-	[\-t \fIterm\fP,...] [\fIpattern\fP]
+.HP
+\fBpkill\fR [\fB\-\fIsignal\fR] [\fB\-fvx\fR] [\fB\-n\fR|\fB\-o\fR] \
+[\fB\-P\ \fIppid\fR,...] [\fB\-g\ \fIpgrp\fR,...] [\fB\-s\ \fIsid\fR,...] \
+[\fB\-u\ \fIeuid\fR,...] [\fB\-U\ \fIuid\fR,...] [\fB\-G\ \fIgid\fR,...] \
+[\fB\-t\ \fIterm\fR,...] [\fIpattern\fR]
 
 .SH DESCRIPTION
 \fBpgrep\fP looks through the currently running processes and lists the
 process IDs which matches the selection criteria to stdout.  All
 the criteria have to match.  For example,
 
-pgrep -u root sshd
+.IP
+$ pgrep \-u root sshd
 
+.PP
 will only list the processes called \fBsshd\fP AND owned by \fBroot\fP.
 On the other hand,
 
-pgrep -u root,daemon
+.IP
+$ pgrep \-u root,daemon
 
+.PP
 will list the processes owned by \fBroot\fP OR \fBdaemon\fP.
 
 \fBpkill\fP will send the specified signal (by default \fBSIGTERM\fP)
@@ -38,18 +43,18 @@
 
 .SH OPTIONS
 .TP
-\-c
+\fB\-c\fR
 Suppress normal output; instead print a count of matching processes.
 .TP
-\-d \fIdelimiter\fP
+\fB\-d \fIdelimiter\fP
 Sets the string used to delimit each process ID in the output (by
 default a newline).  (\fBpgrep\fP only.)
 .TP
-\-f
+\fB\-f\fR
 The \fIpattern\fP is normally only matched against the process name.
-When \-f is set, the full command line is used.
+When \fB\-f\fR is set, the full command line is used.
 .TP
-\-g \fIpgrp\fP,...
+\fB\-g \fIpgrp\fP,...
 Only match processes in the process group IDs listed.  Process group 0
 is translated into \fBpgrep\fP's or \fBpkill\fP's own process group.
 .TP
@@ -57,40 +62,40 @@
 Only match processes whose real group ID is listed.  Either the
 numerical or symbolical value may be used.
 .TP
-\-l
+\fB\-l\fR
 List the process name as well as the process ID. (\fBpgrep\fP only.)
 .TP
-\-n
+\fB\-n\fR
 Select only the newest (most recently started) of the matching
 processes.
 .TP
-\-o
+\fB\-o\fR
 Select only the oldest (least recently started) of the matching
 processes.
 .TP
-\-P \fIppid\fP,...
+\fB\-P \fIppid\fP,...
 Only match processes whose parent process ID is listed.
 .TP
-\-s \fIsid\fP,...
+\fB\-s \fIsid\fP,...
 Only match processes whose process session ID is listed.  Session ID 0
 is translated into \fBpgrep\fP's or \fBpkill\fP's own session ID.
 .TP
-\-t \fIterm\fP,...
+\fB\-t \fIterm\fP,...
 Only match processes whose controlling terminal is listed.  The
 terminal name should be specified without the "/dev/" prefix.
 .TP
-\-u \fIeuid\fP,...
+\fB\-u \fIeuid\fP,...
 Only match processes whose effective user ID is listed.  Either the
 numerical or symbolical value may be used.
 .TP
-\-U \fIuid\fP,...
+\fB\-U \fIuid\fP,...
 Only match processes whose real user ID is listed.  Either the
 numerical or symbolical value may be used.
 .TP
-\-v
+\fB\-v\fR
 Negates the matching.
 .TP
-\-x
+\fB\-x\fR
 Only match processes whose name (or command line if \-f is specified)
 \fBexactly\fP match the \fIpattern\fP.
 .TP
@@ -107,32 +112,40 @@
 .SH EXAMPLES
 Example 1: Find the process ID of the \fBnamed\fP daemon:
 
-unix$ pgrep \-u root named
+.IP
+$ pgrep \-u root named
 
+.PP
 Example 2: Make \fBsyslog\fP reread its configuration file:
 
-unix$ pkill \-HUP syslogd
+.IP
+$ pkill \-HUP syslogd
 
+.PP
 Example 3: Give detailed information on all \fBxterm\fP processes:
 
-unix$ ps \-fp $(pgrep \-d, \-x xterm)
+.IP
+$ ps \-fp $(pgrep \-d, \-x xterm)
 
+.PP
 Example 4: Make all \fBnetscape\fP processes run nicer:
 
-unix$ renice +4 `pgrep netscape`
+.IP
+$ renice +4 `pgrep netscape`
 
 .SH "EXIT STATUS"
+.PD 0
 .TP
-.I "0"
+.I 0
 One or more processes matched the criteria.
 .TP
-.I "1"
+.I 1
 No processes matched.
 .TP
-.I "2"
+.I 2
 Syntax error in the command line.
 .TP
-.I "3"
+.I 3
 Fatal error: out of memory etc.
 
 .SH NOTES
@@ -144,8 +157,8 @@
 itself as a match.
 
 .SH BUGS
-The options \-n and \-o and \-v can not be combined.  Let me know if
-you need to do this.
+The options \fB\-n\fP and \fB\-o\fP and \fB\-v\fP can not be combined.  
+Let me know if you need to do this.
 
 Defunct processes are reported.
 
